### PR TITLE
[WIP] [DO NOT MERGE] Updating deployment path for che-plugin-registry

### PR DESCRIPTION
**Pull request by ibuziuk**
The new deployment path has been introduced in the https://github.com/eclipse/che-plugin-registry/pull/245

### CODE REVIEW

**Comment by amisevsk** [comment]
My bad, I forgot to update this along with that PR.

**Comment by ibuziuk** [comment]
@amisevsk no worries - it is really hard to follow up on everything related to deployment. I belive we can opt for an alternative solution for now - https://github.com/eclipse/che-plugin-registry/pull/261

**Comment by ibuziuk** [comment]
Closing in favour of https://github.com/openshiftio/saas-openshiftio/pull/1375
